pkg/domain/infra/abi/play.go

```diff
@@ -182,6 +182,9 @@ func (ic *ContainerEngine) playKubePod(ctx context.Context, podName string, podY
 	}
 
 	podOpt := entities.PodCreateOptions{Infra: true, Net: &entities.NetOptions{StaticIP: &net.IP{}, StaticMAC: &net.HardwareAddr{}}}
+	podOpt.Net = &entities.NetOptions{
```

Comment:
move this above into the line with eh already declared net options. If I recall correctly `StaticIP` and `StaticMAC` need to be initialized or a nil pointer deref can happen

Reply:
Good catch.